pyproject.toml

```diff
@@ -0,0 +1,54 @@
+[build-system]
+requires = ["flit_core >=3.2,<4"]
+build-backend = "flit_core.buildapi"
+
+[project]
+# Check https://flit.readthedocs.io/en/latest/pyproject_toml.html for all available sections
+name = "ansys-dpf-core"
+version = "0.7.dev0"
```

Comment:
@PProfizi, how should we define ``version`` here? This is following other pyansys repos convention, but I remember we talked about PyDPF-Core using 0.7.Z.dev0, right?

Reply:
@GuillemBarroso @Revathyvenugopal162 Indeed, the version should be 0.7.3.dev0 (we are releasing 0.7.2)

Reply:
Ok i got your point. but usually we will not keep track of the patch releases, 0.7.dev0 or if we have already 0.7 release, then 0.8.dev0 is the usual way of keeping track, so that the patch release is not going to affect the dev version. https://peps.python.org/pep-0440/#developmental-releases. but no offence. ultimately the developers can decide .

Reply:
I'm fine with either. I think it would be nice to be consistent with other repos and use 0.8.dev0, but I am not fully aware of the implications of changing this. @PProfizi, your call.